Add PyBluez to Dockerfile

philhawthorne opened the pull request:
**Description:** Installs PyBluez in Docker images so users using Docker can use the Bluetooth device tracker.

**Related issue (if applicable):** #1830

**Example entry for `configuration.yaml` (if applicable):**

``` yaml
device_tracker:
  - platform: bluetooth_tracker
    track_new_devices: true
    interval_seconds: 5
    consider_home: 180 
```

**Checklist:**

If code communicates with devices, web services, or a:
- [ ] Local tests with `tox` run successfully. **Your PR cannot be merged unless tests pass**
- [ ] New dependencies have been added to the `REQUIREMENTS` variable ([example][ex-requir]).
- [ ] New dependencies are only imported inside functions that use them ([example][ex-import]).
- [ ] New dependencies have been added to `requirements_all.txt` by running `script/gen_requirements_all.py`.
- [ ] New files were added to `.coveragerc`.

If the code does not interact with devices:
- [ ] Local tests with `tox` run successfully. **Your PR cannot be merged unless tests pass**
- [ ] Tests have been added to verify that the new code works.

Adds PyBluez to Dockerfile so people using Docker can run Bluetooth
devices.

philhawthorne committed:
Add PyBluez to Dockerfile

Adds PyBluez to Dockerfile so people using Docker can run Bluetooth
devices

balloob commented on Dockerfile:
```diff
@@ -10,9 +10,12 @@ RUN pip3 install --no-cache-dir colorlog cython
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# Install pybluez for bluetooth tracker
+RUN pip install pybluez
```
Please combine this with the other call to pip below

balloob replied:
Actually, this can cause issues when the latest version that pip installs is different from what the bluetooth tracker expects. So I would like to suggest that we do not install it. It will be automatically when a user uses it - and since the correct packages have been installed, the installation won't fail.

balloob replied:
So let's remove this line.

balloob commented on virtualization/Docker/Dockerfile.test:
```diff
@@ -10,9 +10,12 @@ RUN pip3 install --no-cache-dir colorlog cython
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# Install pybluez for bluetooth tracker
+RUN pip install pybluez
```
Idem.

balloob reviewed (changes requested):
Let's remove the `pip` commands and this is good to go 🐬

philhawthorne committed:
Remove pip install of pybluez

Pybluez will be installed automatically when the Bluetooth device
tracker is enabled

philhawthorne commented:
Thanks @balloob `pip` command removed. Tested and that works :)

balloob commented:
🐬  !
